doc/z-chap2.xml

```diff
@@ -87,6 +87,7 @@
     <#Include Label="JohnsonDigraph">
     <#Include Label="PetersenGraph">
     <#Include Label="GeneralisedPetersenGraph">
+    <#Include Label="KnightsGraph">
```

Comment:
When you write a new piece of documentation, you also have to  explicitly link it include the manual by including it in one of the chapters, i.e. in one of the `doc/z-chap*.xml` files. Sometimes it's difficult to work out where to put your documentation, but in this case it's easy - we put it with the other standard examples.

If you don't do this step, then your documentation doesn't end up in the compiled version of the manual, and you can't access the manual entry in GAP by doing (in this case) `?KnightsGraph`.